refnx/reflect/simulation.py

```diff
@@ -0,0 +1,167 @@
+import MDAnalysis as mda
+import numpy as np
+from refnx.analysis import Parameters
+from refnx.reflect import structure
+
+
+class Simulation:
+    """
+    Determines the scattering length density profile from a simulation output
```

Comment:
`Determines the scattering length density profile along the z-axis of a molecular dynamics output file (PDB format). Lateral averaging occurs in the x-y plane.`

Reply:
Is the file format generic, are there simulation cells for which this code will fail?

Reply:
Done. See discussion later.